backend/corpora/dataset_submissions/app.py

```diff
@@ -38,7 +49,7 @@ def dataset_submissions_handler(s3_event: dict, unused_context) -> None:
         with db_session_manager() as session:
             collection_owner, dataset_id = get_dataset_info(session, parsed["collection_id"], parsed["dataset_id"])
 
-            logger.info(f"{collection_owner=}, {dataset_id=}")
```

Comment:
This still was not getting logged in previous invocations. If you go and look at the lambda logs you only see `[ERROR]` level logs.

I suspect this will not solve our problem wrt lambda logs. I suspect this is something that needs to be configured in terraform, perhaps via a lambda attribute or possibly via an environment variable.

Reply:
Fixed

Reply:
so you just had to manually specify stdout for the log stream? Interesting 🤔 🤷‍♂️